coka/rest-request-validator-bundle/1.0/config/packages/oka_rest_request_validator.yaml

```diff
@@ -0,0 +1,3 @@
+oka_rest_request_validator:
+    exception:
+        enabled: true
```

Comment:
Is this the same as the default value?

Reply:
Hi,
yes it is the same thing as the default variable but I put it more in mind to indicate what needs to be changed according to these needs

Reply:
Hm... okey.